src/architecture/configuration.md

```diff
@@ -92,17 +109,42 @@ The configuration used to create a genesis block.
 - **tcp_reconnect_timeout_max**  
   Maximum timeout (ms) for reconnect attempt.
 
-#### [node.api]
+#### [api]
 
 API configuration parameters.
 
 - **enable_blockchain_explorer**  
-  Enable api endpoints for the blockchain explorer on the public API address.
+  Enable API endpoints for the blockchain explorer on the public API address.
+- **state_update_timeout**  
+  Timeout (ms) to update info about connected peers.
 - **public_api_address**  
   Listen address for public API endpoints.
 - **private_api_address**  
   Listen address for private API endpoints.
 
+#### [whitelist]
+
+[Whitelist](../advanced/network.md#whitelist) parameters.
+
+- **whitelist_enabled**  
+  Enable whitelisting.
+- **whitelisted_peers**  
+  List containing consensus public keys of trusted peers.
+
+#### [mempool]
+
+Configuring queues sizes.
+
+- **events_pool_capacity**  
+  Maximum number of events in the [event queue](../advanced/consensus/specification.md#message-processing).
+
+- **tx_pool_capacity**  
+  Maximum number of transactions in the [pool of unconfirmed transactions](../advanced/consensus/specification.md#pool-of-unconfirmed-transactions).
+
+#### [services_configs]
+
+Service-specific parameters under the keys named [`service_name`](services.md#service-identifiers).
```

Comment:
Please rephrase to "...under the keys named according to `service_name`s of the blockchain services.